tests/nightly/test_np_large_array.py

```diff
@@ -568,6 +502,517 @@ def test_slice_assign():
+    with mx.autograd.record():
+        B = np.fliplr(A)
+        B.backward()
+    assert B.shape == A.shape
+    assert B[0, 1, 0] == 2
+    assert A.grad.shape == A.shape
+    assert A.grad[0, 0, 0] == 1
+
+@use_np
+def test_flipud():
+    A = np.zeros((2, 1, INT_OVERFLOW))
+    A[0, 0, 0] = 2
+    A.attach_grad()
+    with mx.autograd.record():
+        B = np.flipud(A)
+        B.backward()
+    assert B.shape == A.shape
+    assert B[1, 0, 0] == 2
+    assert A.grad.shape == A.shape
+    assert A.grad[0, 0, 0] == 1
+
+@use_np
+def test_full():
+    A = np.full((INT_OVERFLOW, 2), np.array([1, 2]))
+    assert A.shape == (INT_OVERFLOW, 2)
+    assert A[-1, 0] == 1 and A [-1, 1] == 2
+    B = np.full((2, INT_OVERFLOW), 3)
+    assert B.shape == (2, INT_OVERFLOW)
+    assert B[-1, -1] == 3
+
+@use_np
+def test_full_like():
+    A = np.zeros((INT_OVERFLOW, 2))
+    B = np.full_like(A, 2)
+    assert B.shape == A.shape
+    assert B[-1, -1] == 2
+
+@use_np
+def test_comparison_family():
+    def batch_check(funcs, exp):
+        A.attach_grad()
+        for f, e in zip(funcs, exp):
+            with mx.autograd.record():
+                C = f(A, B)
+                C.backward()
+            assert C.shape == A.shape
+            assert (C[0, 0], C[-1, -1]) == e
+            assert A.grad.shape == A.shape
+            assert A.grad[-1, -1] == 0
+    
+    A = np.ones((INT_OVERFLOW, 2))
+    B = np.zeros((INT_OVERFLOW, 2))
+    B[-1, -1] = 1
+    batch_check([np.greater, np.greater_equal, \
+        np.less, np.less_equal, np.equal, np.not_equal], \
+        [(True, False), (True, True), \
+        (False, False), (False, True), (False, True), (True, False)])
+
+@use_np
+def test_lcm():
+    A = np.ones((2, INT_OVERFLOW), dtype='int32')
+    B = np.ones((2, INT_OVERFLOW), dtype='int32')
+    A[-1, -1] = 3
+    B[-1, -1] = 5
+    A.attach_grad()
+    with mx.autograd.record():
+        C = np.lcm(A, B)
+        C.backward()
+    assert C.shape == A.shape
+    assert C[-1, -1] == 15
+    assert A.grad.shape == A.shape
+    assert A.grad[-1, -1] == 0
+
+@use_np
+def test_log_family():
+    def batch_check(funcs, exp):
+        A.attach_grad()
+        for f, e in zip(funcs, exp):
+            with mx.autograd.record():
+                B = f(A)
+                B.backward()
+            assert B.shape == A.shape
+            assert_almost_equal(B[-1, -1], np.array([e[0]]), \
+                rtol=1e-5, atol=1e-5)
+            assert A.grad.shape == A.shape
+            assert_almost_equal(A.grad[-1, -1], np.array([e[1]]), \
+                rtol=1e-5, atol=1e-5)
+
+    A = np.ones((INT_OVERFLOW, 2))
+    A[-1, -1] = 100
+    batch_check([np.log, np.log10, np.log2, np.log1p], \
+        [(4.6051702, 0.01), (2, 0.00434294), \
+        (6.643856, 0.01442695), (4.6151204, 0.00990099)])
+
+@use_np
+def test_expand_dims():
+    A = np.zeros((INT_OVERFLOW))
+    B = np.expand_dims(A, axis=0)
+    C = np.expand_dims(B, axis=2)
+    npx.waitall()
```

Comment:
just check for values in the expanded dimension ?

Reply:
Will do!